apps/webapp/src/app/app.tsx

```diff
@@ -19,8 +23,20 @@ const LogoWrapper = styled(Layout)``;
 
 export const App = () => {
   const { state: auth } = useAuthentication();
+  const theme = useTheme();
+  const themeMode = useContext(ThemeModeContext);
```

Comment:
podobnie jak w provider, muszę zawsze oba? może jak robię useTheme to wraca zarówno theme, jak i zmieniarka theme. wtedy dekonstrukcją wybieram sobie które z nich akurat potrzebuję? bo wygląda trochę jakby konieczność pobierania tych rzeczy parami powodowała przeciekanie abstrakcji